lib/pkg-config.rb

```diff
@@ -579,6 +580,27 @@ def normalize_cflags(cflags)
     normalized_cflags
   end
 
+  def mergeback_flags(flags)
```

Comment:
Could you add a comment that we want to implement pkgconf's `pkgconf_fragment_copy()` compatible behavior here?

https://github.com/pkgconf/pkgconf/blob/pkgconf-2.5.1/libpkgconf/fragment.c#L381-L416